the-collab-lab/tcl-27-smart-shopping-list
# Added Header & UserToken components and collapsible feature

Co-authored-by: Yolanda Haynes <yhaynes@giyo.us>

## Description

Created Header and UserToken component to display the app’s name and the token throughout the app. The token will only display once the user is signed in. Added new dependency in order for the app to use the collapsible feature.

## Related Issue

#13 

## Acceptance Criteria

- App has a header that is displayed throughout the project
- UserToken is displayed throughout the app to allow the user to share the shopping list with others
- List items can collapse/expand to display a more succinct list

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin tc-yh-add-header-user-token` and check that branch out with `git checkout tc-yh-add-header-user-token`.
- Then `npm install` or `yarn to install` the newly added dependency locally and `npm start` or `yarn start` to launch the app.
- Ensure that the app name "TCL-27" is displayed throughout the app.
- Ensure that the current user token is displayed on all pages of the app once the user is logged in and is not displayed if the user is not logged in.
- Ensure that the list items can expand and collapse by clicking on the name of the items in the List view.
